schemas/coordinator.json

```diff
@@ -0,0 +1,211 @@
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "set_nodes",
+            "params": [
+                {
+                    "name": "nodes",
+                    "schema": {
+                        "$ref": "#/components/nodes"
+                    }
+                }
+            ],
+            "description": "Set node addresses and connect to all nodes, to which there is not yet any connection.",
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "get_nodes",
+            "params": [],
+            "description": "Get the nodes known to this Coordinator.",
+            "result": {
+                "name": "result",
+                "schema": {
+                    "$ref": "#/components/nodes"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "set_components",
+            "params": [
+                {
+                    "name": "components",
+                    "description": "List of Components' names.",
+                    "schema": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "required": true
+                }
+            ],
+            "description": "Update Directory with Components of another Coordinator.",
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "get_local_components",
+            "params": [],
+            "description": "Get the locally connected Components.",
+            "result": {
+                "name": "result",
+                "schema": {
+                    "$ref": "#/components/components_list"
+                },
+                "required": true
+            }
+        },
+        {
+            "name": "get_global_components",
+            "params": [],
+            "description": "Get all Components in this LECO network.",
+            "result": {
+                "name": "result",
+                "schema": {
+                    "$ref": "#/components/global_directory"
+                }
+            }
+        },
+        {
+            "name": "clean_addresses",
+            "params": [
+                {
+                    "name": "expiration_time",
+                    "description": "Expiration time of Components and Coordinators in s.",
+                    "summary": "seconds",
```

Comment:
Question: Where do we define the unit: in the method description, parameter description, parameter summary?

I prefer to define it **just once**, lest there are contradicting definitions.

We use mainly description, so I'd place it there.